src/App.jsx

```diff
@@ -11,38 +11,57 @@ import MainLayout from '@/layouts/MainLayout';
 import AppLayout from '@/layouts/AppLayout';
 import { RegisterStaff } from '@/pages/app/RegisterStaff';
 import { AuthProvider, ProtectedRoute } from '@/components/auth-provider';
+import { ClinicCreation } from '@/pages/ClinicCreation';
+import { ClinicaEdicion } from '@/pages/EditClinic';
+import { ClinicaCompletada } from '@/pages/SuccessPage';
 
 function App() {
   return (
     <AuthProvider>
       <ThemeProvider storageKey="vite-ui-theme">
         <Router>
           <Routes>
-            {/* Doing nested routes allows to avoid re-rendering re-used components */}
             <Route path="/" element={<MainLayout />}>
               <Route index element={<Landing />} />
               <Route path="about" element={<About />} />
             </Route>
-            <Route path="/app" element={
-              <ProtectedRoute>
-                <AppLayout />
-              </ProtectedRoute>
-            }>
+            <Route path="/app" element={<AppLayout />}>
               <Route index element={<Home />} />
-              <Route path="register-staff" element={
-                <ProtectedRoute allowedRoles={['admin', 'clinicadmin']}>
-                  <RegisterStaff />
-                </ProtectedRoute>
-              } />
+              <Route path="register-staff" element={<RegisterStaff />} />
             </Route>
-            { /* Routes here have no layout ON PURPOSE */}
             <Route path="/login" element={<Login />} />
             <Route path="/logout" element={<Logout />} />
             <Route path="/verify-2fa" element={<Verify2FA />} />
+
+            {/* Rutas relacionadas con Clínicas */}
+            <Route
+              path="/clinics/add"
+              element={
+                <ProtectedRoute allowedRoles={['admin', 'clinicadmin']}>
+                  <ClinicCreation />
+                </ProtectedRoute>
+              }
+            />
+            <Route
+              path="/clinics/:id/edit"
+              element={
+                <ProtectedRoute allowedRoles={['admin', 'clinicadmin']}>
+                  <ClinicaEdicion />
+                </ProtectedRoute>
+              }
+            />
+            <Route
+              path="/clinics/success"
+              element={
+                <ProtectedRoute allowedRoles={['admin', 'clinicadmin']}>
+                  <ClinicaCompletada />
+                </ProtectedRoute>
+              }
+            />
```

Comment:
Correct the bad conflict solve and put clinic pages inside app. Also move the .jsx pages to /pages/app